status-im/status-react
# feature #6026 - USDC token

fixes #6026 

### Summary:

Added USDC token


### Steps to test:
- On Mainnet, get, send and receive USDC token
- Check if it appears ok in history
